### PR TITLE
Modify the amount to diamond and emerald generators

### LINKED ISSUE #136: Modify the amount of emeralds and diamonds that the generators give
**Is your feature request related to a problem? Please describe.**
I would like to modify the amount of emeralds and diamonds that the generators give in each cycle of time.

**Describe the solution you'd like**
To be able to modify the amount of emeralds and diamonds that the generators give, as in the gold and iron generators

### PR DESCRIPTION
**Pull request by reussy**
I remove the check **isFirstTime** because it never overwrites existing values, so when creating the file or checking that the new paths are missing it will always add them without affecting the others
The new paths are created by default with 1

Closes #136

### CODE REVIEW

File: bedwars-plugin/src/main/java/com/andrei1058/bedwars/configuration/GeneratorsConfig.java [hunk 1]
```diff
@@ -30,7 +30,7 @@
     public GeneratorsConfig(Plugin plugin, String name, String dir) {
         super(plugin, name, dir);
 
-        if (isFirstTime()) {
+        //if (isFirstTime()) {
```
Comment (ajgeiss0702) [inline bedwars-plugin/src/main/java/com/andrei1058/bedwars/configuration/GeneratorsConfig.java]: If you are going to remove this, please actually remove it, then fix the indenting for the lines below.

**Review by ajgeiss0702 — CHANGES_REQUESTED** [review]

**Review by ajgeiss0702 — APPROVED** [review]
Looks good

**Comment by andrei1058** [comment]
I'm worrying the thing you removed was there for a reason, but I can't check it right now because reviewing on mobile is not the greatest :(

**Comment by reussy** [comment]
I have checked that everything was fine before committing
Checking the code only verifies if it is the first time you start the plugin (that the files do not exist) but it does not do much more than that

**Comment by ajgeiss0702** [comment]
I just double-checked, it looks like the isFirstTime method is unnecessary. The method was not used for the main config, and we know the main config works fine.

**Review by andrei1058 — APPROVED** [review]

**Comment by andrei1058** [comment]
Thank you very much!

**Pull request merged**